tests/components/UserActions/ClickedDocumentList.spec.ts

```diff
@@ -270,6 +271,43 @@ describe('ClickedDocumentList', () => {
         expect(mock.cmp.disabled).toBe(true);
     });
 
+    it('should log a userActionDocumentClick event when a result link in the template is clicked', async () => {
```

Comment:
this test is a bit girthy but if we aren't reusing any of this elsewhere i guess it doesn't make much sense to extract any of it either.